### DISCUSSION TITLE
Why am I getting rate limited by Dockerhub when pulling images?

### DISCUSSION THREAD

**Discussion by guisellaDes** [kurtosis-tech/kurtosis#1272] [Q&A] [answered]
Why am I getting rate limited by Dockerhub when pulling images?

**Comment by leeederek** [comment, marked as answer]
Hi @guisellaDes!

If you are getting rate limited by Dockerhub when pulling images, it likely means you have exceeded the [limits set by Docker](https://docs.docker.com/docker-hub/download-rate-limit/).
